tgui/packages/tgui/components/Section.tsx

```diff
@@ -12,6 +12,8 @@ import { BoxProps, computeBoxClassName, computeBoxProps } from './Box';
 type SectionProps = BoxProps & {
   className?: string;
   title?: InfernoNode;
+  titleStyle?: String | undefined;
+  textStyle?: String | undefined;
```

Comment:
No point in `undefined` type since the entire field is optional.

Reply:
I need to talk more about these extensions, because I'm not sure if we need them (you already wrote a custom section for departments), and additions like these are not cheap and are going to slow down all UIs.

Reply:
Oh, yea, I think I forgot to revert that before committing, cause I figured it was a terrible idea for a single menu that uses it! I'll fix that when I'm able!